fdbclient/include/fdbclient/MetaclusterManagement.actor.h

```diff
@@ -1544,6 +1544,28 @@ Future<Void> deleteTenant(Reference<DB> db, int64_t id) {
 	return Void();
 }
 
+template <class Transaction>
+Future<std::vector<std::pair<TenantName, int64_t>>> listTenantIdsTransaction(Transaction tr,
+                                                                             TenantName begin,
+                                                                             TenantName end,
+                                                                             int limit) {
+	tr->setOption(FDBTransactionOptions::READ_SYSTEM_KEYS);
+	auto future = ManagementClusterMetadata::tenantMetadata().tenantNameIndex.getRange(tr, begin, end, limit);
+	return fmap([](auto f) -> std::vector<std::pair<TenantName, int64_t>> { return f.results; }, future);
+}
+
+template <class DB>
+Future<std::vector<std::pair<TenantName, int64_t>>> listTenantIds(Reference<DB> db,
```

Comment:
List tenants should be the interface that returns IDs. If we want to have functions in here for getting the tenant metadata, we can call that something like `listTenantMetadata`